src/checks.js

```diff
@@ -29,16 +29,30 @@ function CheckFailed(message, severity) {
 }
 
 function getHint(allowedParents) {
+	allowedParents.sort()
 	switch (allowedParents.length) {
 		case 0:
 			return ""
 		case 1:
 			return `Parent must be "${allowedParents[0]}".`
 		default:
-			return `\nValid parent blocks: ${makeQuotedList(allowedParents)}`
+			let hint = `\nMust be within one of ${makeQuotedList(allowedParents)}`
```

Comment:
If the phrase 'Parent must be ...' is changed here to 'Must be within' in light of the actual parent being `environment` (https://github.com/avenga/couper-vscode/pull/92), then the phrase for `case 1` should be changed, too. E.g.

```hcl
saml "..." {
  environment "prod" {
    api {
    }
  }
}
```
Block "api" is invalid within "saml". _Parent_ must be "server".